docs/2 Run a Blockchain/1 Start a Blockchain.md

```diff
@@ -1,4 +1,33 @@
-# Run a blockchain
+# Start a Blockchain
 
-(this old content seems to introduce a directory structure, do we want to talk about that structure? no edits yet)
-When you start the blockchain app with the `starport serve`, the blockchain folder user homefolder `~/.myappd` (the name of your app with a `d` for `daemon` attached) and initiate your blockchain with the genesis file, located under `~/.myappd/config`. The second folder you can find in the `~/.myappd` folder is `data` - this is where the blockchain will write the consecutive blocks and transactions. The other folder created is the `~/.myappcli` folder, which contains a configuration file for your current command line interface, such as `chain-id`, output parameters such as `json` or `indent` mode. If you want to make sure all of your data from the blockchain setup is deleted, make sure to remove the `~/.myappd` and `~/.myappcli` folder.
+One of the first things you might want to do is start your blockchain. You can start your blockchain with the `starport serve` command.
+
+The command does the following:
+
+* Installs dependencies
+* Imports state (if possible)
+* Builds protocol buffer files
+* Generates JS, TS and Vuex clients (optional)
+* Builds blockchain's binary
+* Creates accounts
+* Initializes a blockchain node
+* Starts processes
+  * Tendermint RPC
+  * Cosmos SDK API
+  * Faucet (optional)
+  * Welcome screen
+  * Web scaffold (optional)
+* Watches for file changes and restarts
+* Exports state
+
+Optional `--config` flag: specify a custom configuration file. By default value is `config.yml`. Useful for launching two blockchains on the same machine, from the same source code using two different configuration files.
+
+Optional `--reset-once` flag: resets state once. By default Starport will try to import existing state and resume the blockchain (state persistence). If that fails or if you want to initialise a blockchain from an empty state, use `--reset-once`.
+
+Optional `--force-reset` flag: reset state on every file change, state persistence is turned off and Starport will not try to import state.
+
+Optional `--rebuild-proto-once` flag: forces Starport to perform code generation from proto files for both custom and third-party modules. Use in combination with `--reset-once`. By default Starport scaffolds files generated from Cosmos SDK standard proto files, instead of generating them on the fly. Useful if you've scaffolded a blockchain with a previous version of Starport or you've upgraded Cosmos SDK version and you want to make sure that code generation is performed for all modules.
+
+Optional `--verbose` flag: enters verbose (detailed) mode with extensive logging.
+
+Optional `--home` flag: specify a custom home directory.
```

Comment:
Three situations. If nothing is specified, (1) Starport uses the project's name as a directory name for the home data dir. For example, for `starport app github.com/cosmos/gaia` it will create `~/.gaia/`. If (2) `init.home` is specified in `config.yml` t will use that. If (3) `--home` is passed as an argument to `serve`, this value will be used instead of the one in config or instead of the default one.

Reply:
The change has become outdated, so we'll have to apply it manually.